extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -141,6 +141,15 @@ describe('amp-analytics', function() {
     return waitForSendRequest(analytics, 100);
   }
 
+  function clearIframePing(config) {
```

Comment:
added. What's the general policy on jsdocs for tests? The function is literally unsetting (clearing) iframPing property in the config.